addons/l10n_au/views/report_invoice.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <template id="report_invoice_document" inherit_id="account.report_invoice_document">
+        <xpath expr="//div[@class='page']/h2" position="replace">
+            <h2>
+                <span t-if="o.move_type == 'out_invoice' and o.state == 'posted'">Tax Invoice</span>
+                <span t-if="o.move_type == 'out_invoice' and o.state == 'draft'">Draft Tax Invoice</span>
+                <span t-if="o.move_type == 'out_invoice' and o.state == 'cancel'">Cancelled Tax Invoice</span>
+                <span t-if="o.move_type == 'out_refund'">Tax Credit Note</span>
+                <span t-if="o.move_type == 'in_refund'">Tax Vendor Credit Note</span>
+                <span t-if="o.move_type == 'in_invoice'">Tax Vendor Bill</span>
+                <span t-if="o.name != '/'" t-field="o.name"/>
```

Comment:
You probably can use t-elif

Reply:
True. It was copied from the base template literally, but it might be a little more efficient. Changed it now.
I also have a task to modify the base view, so I'll change it there afterwards too ;)